packages/transactions/src/OutboundShipment/ListView.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { FC, useState } from 'react';
```

Comment:
This has just been shifted from the original file. Slight change to the query to use a variable and added the new shipment button